src/riak_test_escript.erl

```diff
@@ -297,26 +297,45 @@ run_test(Test, Outdir, TestMetaData, Report, HarnessArgs, NumTests) ->
         1 -> keep_them_up;
         _ -> rt:teardown()
     end,
-    CoverageFile = rt_cover:maybe_export_coverage(Test, CoverDir, erlang:phash2(TestMetaData)),
```

Comment:
No need to break up these long lines - last I heard, anything below 100 characters is fair game. (I know some people will say 80, but I believe the consensus was that the 80 char limit is a holdover from old line printers and 80x25 terminals, and is unnecessarily short for today's monitors.)